R/flywire-skeleton.R

```diff
@@ -255,15 +281,20 @@ skeletor <- function(segments = NULL,
                      n_rays = 20,
                      projection = c("sphere", "tangents"),
                      fallback = "knn",
+                     waves=2,
+                     step_size=1,
                      sampling_dist=500,
                      cluster_pos = c("median", "center"),
                      shape_weight = 1,
                      sample_weight = 0.1,
+                     inv_dist = 100,
```

Comment:
@alexanderbates can you document this new argument?

Reply:
Ah, misnamed. Fixed